main.js

```diff
@@ -16,26 +16,54 @@ const removeClass = (el, oldClass) => {
   const inputField = document.getElementsByTagName('input')[0];
   let typeTimeout;
   const TYPE_INTERVAL = 10000;
+  const siteContainer = document.getElementsByTagName('body');
 
   inputField.addEventListener("input", (event) => {
     const value = event.target.value;
     if (typeTimeout) window.clearTimeout(typeTimeout);
 
     if (value) {
       typeTimeout = delay(matchKeyword(value), TYPE_INTERVAL);
+    } else {
```

Comment:
I've DEFINITELY noticed it. I'll play with increasing/decreasing `TYPE_INTERVAL` and see if that resolves the issue.